indico/modules/events/registration/models/items.py

```diff
@@ -49,7 +49,7 @@ class PersonalDataType(IndicoIntEnum):
     """Description of the personal data items that exist on every registration form."""
 
     __titles__ = [None, 'Email Address', 'First Name', 'Last Name', 'Affiliation', 'Title', 'Address',
-                  'Phone Number', 'Country', 'Position']
+                  'Phone Number', 'Country', 'Position', 'Picture Id']
```

Comment:
What's a "Picture Id"? I think this should be just "Picture".

Reply:
Done.